crates/autopilot/src/run_loop.rs

```diff
@@ -657,7 +658,7 @@ struct Metrics {
     #[metric(labels("driver", "result"))]
     reveal: prometheus::IntCounterVec,
 
-    /// Tracks the result of driver `/settle` requests.
+    /// Tracks the times and results of driver `/settle` requests.
```

Comment:
I'm a bit worried that given the other metric names an unsuspecting developer may still interpret this metric as number of settle calls (instead of the time). Maybe we can rename this metric to reflect this?

Also times are normally recorded using histograms, but it may be overkill here.

Reply:
Sure, renaming makes sense.
> Also times are normally recorded using histograms, but it may be overkill here.

I was considering using histograms but I'd prefer to have the better accuracy of the counter approach. Also IIRC it's easier to write grafana queries using counters than using histograms (if you don't want to create a heatmap).